db/upgrade.php

```diff
@@ -157,11 +157,32 @@ function xmldb_surveypro_upgrade($oldversion) {
     }
 
     if ($oldversion < 2016100601) {
-        $DB->delete_records('surveypro_answer', array('content' => '@@_ANINDB_@@'));
+        $where = $DB->sql_compare_text('content').' = :content';
```

Comment:
By exporting and reimporting data you may see some surveyproanswer.cotentformat == '@@_ANINDB_@@'
I did this for a 120 elements long surveypro with about 500 answer and I found a lot of occurrences of this problem.